packages/common/src/sites/_internal/_ensure-event-list-card.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  IModel,
+  IDraft,
+  getProp,
+  cloneObject,
+  deleteProp,
+  setProp,
+} from "../..";
+
+/**
+ * Add telemetry config object
+ * @private
+ * @param {object} model Site Model
+ * @returns {object}
+ */
+export function _ensureEventListCard<T extends IModel | IDraft>(model: T): T {
```

Comment:
Minor, but this migration name suggests event-list-card configs don't exist but we'll make one/more exist, which isn't quite what we're doing. It may make more sense to rename this function to something like `_migrateEventListCardConfigs`.